themes/src/main/resources/theme/emeritus-theme/login/template.ftl

```diff
@@ -51,9 +51,14 @@
 <div class="${properties.kcLoginClass!}">
     <div id="kc-header" class="${properties.kcHeaderClass!}">
           <div id="kc-header-wrapper" class="${properties.kcHeaderWrapperClass!}">
-        <img id="kc-header-logo" src="https://s40424.pcdn.co/in/wp-content/uploads/2022/02/emeritus-logo.png" alt="company"/>
+        <img onclick="redirectToInsights()" id="kc-header-logo" src="https://s40424.pcdn.co/in/wp-content/uploads/2022/02/emeritus-logo.png" alt="company"/>
         </div>
     </div>
+<script>
+   function redirectToInsights() {
+        window.location.href = 'https://insights-fe-staging-b1fc39a5bdb9.herokuapp.com/';
```

Comment:
it should be coming from env

Reply:
as of now it is not possible so we could hard code to prod url